fabtests/functional/rdm_stress.c

```diff
@@ -921,29 +1144,32 @@ int main(int argc, char **argv)
 			ft_parseinfo(op, optarg, hints, &opts);
 			break;
 		case 'u':
-			ctrlfile = strdup(optarg);
+			ctrlfile = optarg;
 			break;
 		case '?':
 		case 'h':
 			ft_csusage(argv[0], "An RDM endpoint error stress test.");
+			FT_PRINT_OPTS_USAGE("-u <test control file>", "Sample in fabtests/test_configs/ofi_rxm/stress.json");
```

Comment:
line length is way too long here

Reply:
Yeah, this one is tough.  Don't like to break up a string into 2.  Is the help text too long?  Remove the reference to stress.json?  Would anyone know where it may be?

Reply:
Break after the `,`,  and align the second string with first one.